.travis.yml

```diff
@@ -24,5 +22,5 @@ script:
   - go test -v -race ./...
   - |
     if [[ $TRAVIS_GO_VERSION == '1.13.x' && $TRAVIS_OS_NAME == 'linux' ]]; then
-      golangci-lint run
+      golangci-lint run --exclude-use-default=false
```

Comment:
Should we just set this in the config file so that people get the linters with a normal `golangci-lint run`?

Reply:
Oh yeah, good idea. Thanks!